packages/suite/src/components/wallet/WalletLayout/AccountBanners/TaprootBanner.tsx

```diff
@@ -49,26 +36,28 @@ const Divider = styled.div`
 `;
 
 interface TaprootBannerProps {
-    onClose: () => void;
+    account?: Account;
 }
 
-export const TaprootBanner = ({ onClose }: TaprootBannerProps) => {
+export const TaprootBanner = ({ account }: TaprootBannerProps) => {
     const theme = useTheme();
-    return (
-        <StyledCard data-test="@accounts/empty-account/taproot-account">
-            <Heading>
-                <Title>
-                    <Translation id="TR_TAPROOT_BANNER_TITLE" />
-                </Title>
-                <Button
-                    data-test="@accounts/empty-account/taproot-account/close"
-                    variant="tertiary"
-                    onClick={onClose}
-                >
-                    <Translation id="TR_GOT_IT" />
-                </Button>
-            </Heading>
+    const { taprootBannerClosed } = useSelector(state => state.suite.flags);
+    const dispatch = useDispatch();
+
+    const isVisible = !taprootBannerClosed && account && getBip43Type(account.path) === 'bip86';
+
+    if (!isVisible) {
+        return null;
+    }
 
+    const closeTaprootBanner = () => dispatch(setFlag('taprootBannerClosed', true));
+
+    return (
+        <CloseableBanner
+            onClose={closeTaprootBanner}
+            title={<Translation id="TR_TAPROOT_BANNER_TITLE" />}
+            color={theme.TYPE_GREEN}
```

Comment:
Ideally please use `Box` component. 
Also we don't use outdated color tokens with uppercase. It should be probably `backgroundPrimaryDefault`

Reply:
https://github.com/trezor/trezor-suite/pull/11099#discussion_r1485911502